File: product-service/src/main/java/com/ezbuy/productservice/client/properties/AuthProperties.java
package com.ezbuy.productservice.client.properties;

import io.hoangtien2k3.reactify.client.properties.WebClientProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component("authProperties")
@ConfigurationProperties(prefix = "client.auth", ignoreInvalidFields = true)
@Data
@AllArgsConstructor
public class AuthProperties extends WebClientProperties {}

File: notification-service/src/main/kotlin/com/ezbuy/notificationservice/client/properties/AuthProperties.kt
package com.ezbuy.notificationservice.client.properties

import io.hoangtien2k3.reactify.client.properties.WebClientProperties
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component
import lombok.AllArgsConstructor

@Component("authProperties")
@ConfigurationProperties(prefix = "client.auth", ignoreInvalidFields = true)
@AllArgsConstructor
class AuthProperties : WebClientProperties()
